Client/Frontend/Library/ReaderPanel.swift

```diff
@@ -268,7 +272,7 @@ class ReadingListPanel: UITableViewController, LibraryPanel {
         welcomeLabel.adjustsFontSizeToFitWidth = true
         welcomeLabel.snp.makeConstraints { make in
             make.centerX.equalToSuperview()
-            make.top.equalToSuperview().offset(50)
+            make.top.equalToSuperview().offset(150)
```

Comment:
what is this change for?

Reply:
For some reason, the vertical placement of the empty state content was not positioned correctly after changing around the view hierarchy.